lms/djangoapps/course_api/blocks/tests/test_views.py

```diff
@@ -1,27 +1,28 @@
 """
 Tests for Blocks Views
 """
-import ddt
-
 from datetime import datetime
 from unittest import mock
 from unittest.mock import Mock
 from urllib.parse import urlencode, urlunparse
 
+import ddt
 from completion.test_utils import CompletionWaffleTestMixin, submit_completions_for_testing
 from django.conf import settings
 from django.urls import reverse
 from opaque_keys.edx.locator import BlockUsageLocator, CourseLocator
+from rest_framework.utils.serializer_helpers import ReturnList
 
 from common.djangoapps.student.models import CourseEnrollment
 from common.djangoapps.student.roles import CourseDataResearcherRole
 from common.djangoapps.student.tests.factories import AdminFactory, CourseEnrollmentFactory, UserFactory
-from openedx.core.djangoapps.discussions.models import (
-    DiscussionsConfiguration,
-    Provider,
+from openedx.core.djangoapps.discussions.models import DiscussionsConfiguration, Provider
+from xmodule.modulestore.tests.django_utils import \
+    SharedModuleStoreTestCase  # lint-amnesty, pylint: disable=wrong-import-order
+from xmodule.modulestore.tests.factories import (  # lint-amnesty, pylint: disable=wrong-import-order
+    BlockFactory,
+    ToyCourseFactory
```

Comment:
This seems like a weird formatting change, why remove the parens on the first entry and add them to the last one?

Reply:
@feanil I had run iSort on this file but yes it seems weird with / I have manually updated the line and added parentheses. Other than this there were few unused imports do you want me to put those back as well?

Reply:
I think dropping the unused imports is great.  This is fine to leave in as-is if this is just weird isort behavior.